docs/getting-started-acmecorp/create-policy.mdx

```diff
@@ -86,6 +92,10 @@ First, click on the "Policies" tab and click the "Create an instance" button:
 
 <img src="/getting-started-acmecorp/create-instance-button.png" width="170" />
 
+Next, you'll be asked to select between creating the policy from a sample image or from your own policy image - choose "...from your own policy image".
```

Comment:
Isn't there a missing step here where you ask the user to navigate to the Policies tab?

Reply:
Line 91 says: 
```
First, click on the "Policies" tab and click the "Create an instance" button:
```